syntax = "proto3";

package mprpc;

message mpRpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}